File: 2114.maximum-number-of-words-found-in-sentences.java
/*
 * @lc app=leetcode id=2114 lang=java
 *
 * [2114] Maximum Number of Words Found in Sentences
 */

// @lc code=start
class Solution {
    public int mostWordsFound(String[] sentences) {
        int i = 0;
    for (String s : sentences) {
      int count = 1;
      for (char c : s.toCharArray()) {
        if (c == ' ') {
          count++;
        }
        i = Math.max(i, count);
      }
    }
    return i;
    }
}
// @lc code=end

File: 2114.maximum-number-of-words-found-in-sentences.py
#
# @lc app=leetcode id=2114 lang=python
#
# [2114] Maximum Number of Words Found in Sentences
#

# @lc code=start
class Solution(object):
    def mostWordsFound(self, sentences):
        """
        :type sentences: List[str]
        :rtype: int
        """
        m = 0
        for i in sentences:
            c = 1
            for j in i:
                if j == ' ':
                    c += 1
            m = max(m,c)
        return m
        
# @lc code=end

